test/fuzz/lua/CMakeLists.txt

```diff
@@ -102,6 +102,7 @@ string(JOIN ";" LUZER_TESTS
   msgpack_itv_test.lua
   net_box_call_test.lua
   pickle_unpack_test.lua
+  test_engine.lua
```

Comment:
This will result in the following available tests:
```
  Test #2189: test/fuzz/lua/test_engine.lua:memtx
  Test #2190: test/fuzz/lua/test_engine.lua:vinyl
  Test #2308: test_engine
```

As I mentioned in https://github.com/tarantool/tarantool/pull/11301#issuecomment-3522641967 -- it would be nice to define the luzer tests with the full names as well, so we got name clashing here. Also, it would be nice to support different space engines by luzer as well.